challenges/objects-arrays.js

```diff
@@ -52,6 +79,13 @@ const graduates = [
 
 Once you have the new array created, sort the universities alphabetically and log the result. */
 const universities = [];
+
+for (i = 0; i < graduates.length; i++) {
+  graduates[i].university;    
+  universities.push(graduates[i].university);
+  universities.sort();
+}
```

Comment:
Excellent for loop. Remember that you might be able to do the same thing with the `map` array method!